packages/react/src/__tests__/storyshots.test.js

```diff
@@ -9,6 +9,8 @@ import initStoryshots from '@storybook/addon-storyshots';
 import { mount } from 'enzyme';
 import path from 'path';
 
+process.env = Object.assign(process.env, { DDS_FLAGS_ALL: true });
```

Comment:
Is there other tests that relies on feature flags? Wondering about `--runInBand` scenario (where the process of all test cases are shared).